app/api/views/briefs.py

```diff
@@ -148,18 +189,39 @@ def download_brief_response_file(brief_id, supplier_code, slug):
 @login_required
 def post_brief_response(brief_id):
     brief_response_json = get_json_from_request()
-
     supplier, brief = _can_do_brief_response(brief_id)
+    # TODO fix frontend to only send arrays for these three properties
+    array_fields = ['essentialRequirements', 'niceToHaveRequirements', 'attachedDocumentURL']
+    try:
+        for af in array_fields:
```

Comment:
don't like the look of this...

Reply:
removed with removal of brief_response_answer